tests/unit/models/test_evc_deploy.py

```diff
@@ -1846,3 +1889,14 @@ def test_get_priority(self):
 
         epl_value = EVC.get_priority(None)
         self.assertEqual(epl_value, EPL_SB_PRIORITY)
+
+    def test_set_flow_table_group_id(self):
+        """Test set_flow_table_group_id"""
+        self.evc_deploy.table_group = {"epl": 3, "evpl": 4}
```

Comment:
@Alopalao this test case loops good, could you replace the `self.assert.*` with `assert` statements? Even though this is still in a `unittest.TestCase` test suite, since it's run by `pytest` you can still also use assert without issues, that way we also don't add technical debt in the codebase.

Reply:
Right, changed it.